custom_components/waste_collection_schedule/waste_collection_schedule/source/region_gotland_se.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from datetime import datetime
+import json
+from urllib.parse import urlencode
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Region Gotland"
+DESCRIPTION = "Source for Region Gotland waste collection."
+URL = "https://edpfuture.gotland.se"
+TEST_CASES = {
+    "Dummy": {"uprn": "0000000000"},
```

Comment:
This doesn't look like a valid uprn. Please provide at least one valid test case which can be test with `test_sources.py`

Reply:
That's correct. Since the uprns are personal there's no way for me to provide one for the test case. I've tested it with my own personal one to make sure the code works.

Reply:
This is the result from running the test script with my personal uprn:

```
λ python custom_components\waste_collection_schedule\waste_collection_schedule\test\test_sources.py -s region_gotland_se -i -l
Testing source region_gotland_se ...
  found 2 entries for Dummy
    2023-01-25: Restavfall [mdi:trash-can]
    2023-01-25: Matavfall [mdi:leaf]
```

Reply:
You don't have to provide your personal uprn. Is there a public webpage which can be used to get the uprn of e.g. a public building or a restaurant? 
Without a valid test case the code cannot be maintained because we don't know whether it is broken. Maybe you can guess another uprn???

Reply:
I managed to get a hold of a working test uprn so that solves the issue! :)